spec/index.adoc

```diff
@@ -1675,27 +1675,31 @@ It is important to note the following for the usage of the `POST` method:
 
 For security reasons, only HTTPS connections are allowed (URLs must start with `https://`). The IC uses industry-standard root CA lists to validate certificates of remote web servers.
 
-Each request can specify the maximal expected size for the response from the remote HTTP server. The upper limit on the size of a response defaults to `2MiB` if no maximal size value is specified.
+The *size* of an HTTP request or response is the total number of bytes representing the names and values of HTTP headers and the HTTP body. Each request can specify a maximal size for the response from the remote HTTP server. The upper limit on the maximal size for the response defaults to `2MB` (`2,000,000B`) if no maximal size value is specified.
```

Comment:
If someone specifies 5MB it will again default to 2MB which is not very clear from this text.

Reply:
Duplicate.